master/master.cfg

```diff
@@ -261,6 +264,21 @@ test_factory.addStep(ShellCommand(
     logfiles={"console" : { "filename" : "console.log", "follow" : False }},
     decodeRC={0 : SUCCESS, 1 : FAILURE, 2 : WARNINGS, 3 : SKIPPED },
     description=["zfsstress"], descriptionDone=["zfsstress"]))
+test_factory.addStep(Git(repourl=zfs_fedora_installer_repo,
+    workdir="build/tests/zfs-fedora-installer",
+    mode="full", method="clobber", codebase="zfs-fedora-installer",
+    logEnviron=False, getDescription=True,
+    description=["cloning"], descriptionDone=["cloned"]))
+test_factory.addStep(ShellCommand(
+    workdir="build/tests/zfs-fedora-installer",
+    env={'PATH' : bin_path + ":" + zfs_path},
+    command=["runurl", bb_url + "bb-test-bootable-image.sh"],
+    haltOnFailure=False, maxTime=14400, sigtermTime=30, logEnviron=False,
+    logfiles={"console" : { "filename" : "console.log", "follow" : False },
+              "log"     : { "filename" : "output.log",  "follow" : True }},
+    decodeRC={0 : SUCCESS, 1 : FAILURE, 2 : FAILURE, 5 : FAILURE, 9 : FAILURE, 18 : SKIPPED },
```

Comment:
This was in regards to the decodeRC values.  It's not a very complicated script so I'd suggest sticking to the default exit values used by the other scripts.